supernode/daemon/mgr/ha/etcd_tool.go

```diff
@@ -0,0 +1,136 @@
+package ha
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/dragonflyoss/Dragonfly/supernode/config"
+
+	"go.etcd.io/etcd/clientv3"
+	"go.etcd.io/etcd/mvcc/mvccpb"
+)
+
+//EtcdMgr is the struct to manager etcd
+type EtcdMgr struct {
+	config            clientv3.Config
+	client            *clientv3.Client
+	leaseTTL          int64
+	leaseKeepAliveRsp <-chan *clientv3.LeaseKeepAliveResponse
+	hostIP            string //i  will try to use a already existing config struct to  replace it
```

Comment:
Not use `i will do something`. That looks casual.  And you can write comments looks like as follows:

```
TODO(yunfeiyangbuaa):  Use an already existing config struct to  replace it.
```